### ISSUE TITLE
Image alt text should be more descriptive for screen readers

### ISSUE THREAD

**Issue by steverep** [davidsneal/simplesharebuttons#1]
The alt attributes of each share button simply name the social site (e.g. Facebook button is alt="Facebook").  This means that a visually-impaired person using a screen reader could easily misinterpret the link.  The alt attribute should be more descriptive of the action occurring by following the link (e.g. "Share on Facebook", "Share on Google+", "Flattr the author", etc.).

This is a fairly simple change so hopefully you can accommodate in the next release so I can use this plugin on my site.  Thanks!

**Comment by davidsneal** [comment]
Hi Steve, I'll ensure this is in place for the next release, thanks for the suggestion!

**Comment by steverep** [comment]
Great!  Better yet, let me control it in the Advanced section ;).

**Comment by davidsneal** [comment]
Hi Steve. I gave that a little thought but it would mean more strain on servers, space in databases and a more complicated control panel. I have released the update to GitHub if you wish to update now, but I shall make it part of the next official release too for 3.5.

Thanks!

**Comment by steverep** [comment]
Thanks!

P.S. I was only half-serious in giving user's control, since some might miss the point and make it less accessible.  You've forced it to be more accessible - hard to complain about that.  Cheers.
